quantum/keymap_extras/keymap_bepo.h

```diff
@@ -168,6 +168,7 @@
 #define BP_UGRV ALGR(BP_U)    // Ù
 #define BP_DIAE ALGR(BP_I)    // ¨ (dead)
 #define BP_EURO ALGR(BP_E)    // €
+#define BP_TAPO ALGR(BP_COMM) // ’
```

Comment:
In fact, U+2019 is `RIGHT SINGLE QUOTATION MARK`, which is already present here as `S(ALGR(BP_X))`. So there is no need to duplicate the key, though it may be better to replace that one with this.

Reply:
So what is the policy when there is several combination for the same symbol?

As U+2019 and U+27 are swapped between ALGR(KC_G) and KC_N on Bépo v1.1, this makes a difference. I would like my keymap to use ALGR(KC_G) (or ALGR(BP_COMM)), not S(ALGR(BP_X)).

Reply:
A little bit of context, here is my keymap:

http://www.keyboard-layout-editor.com/#/gists/0003a17e7c0fabf3d9c200fd3c02d23b

or before rebasing on recent QMK : https://git.ni.fr.eu.org/nicolas/qmk_firmware.git/tree/keyboards/splitty/keymaps/ni/keymap.c?h=splitty

The idea is that I use the NAV layer for ALTGR on the right side and NAV on the left side as I only use dead greek on the left side anyway. I also have a problem with BP_NUSL which is now defined as BP_SLSH, pressing NAV-SHIFT-ê, I get a 9 instead of a ^.

Reply:
In this case these files would need to be split into versions for BEPO 1.0 and 1.1 (and any other versions as necessary).